src/insertCss.js

```diff
@@ -87,7 +95,7 @@ function insertCss(styles, options = {}) {
     } else {
       const blob = new Blob([
         css + '\n/*# sourceMappingURL=data:application/json;base64,' +
-        btoa(unescape(encodeURIComponent(JSON.stringify(sourceMap)))) + ' */',
```

Comment:
[`unescape()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape) is deprecated